lib/middleware/get-parse-quotes.js

```diff
@@ -0,0 +1,21 @@
+const fetch = require('cross-fetch');
+
+module.exports = async function getParsedQuotes() {
```

Comment:
This is great, but organization-wise would make more sense to be kept in your services folder. Remember your services are basically where you are doing anything unique to your particular app... like combining/munging data from multiple APIs.